### ISSUE TITLE
Issues with temperature tray text style

### ISSUE THREAD

**Issue by VadimAspirin** [FelixdelasPozas/TrayWeather#143]
1. Temperature text size varies depending on the value:

  ![image](https://user-images.githubusercontent.com/22714352/216833204-c4c4a938-06fd-4e99-998d-a906f54f685a.png)

   It seems that this heterogeneity spoils the style of the system tray. I'm not sure if this was originally intended, but it would be nice to have a setting that fixes the size.

2. As @obuw said [here](https://github.com/FelixdelasPozas/TrayWeather/issues/99#issuecomment-1314994730), scaling the font makes it fuzzy:

   > The font size is by default a little bigger than the other stuff in the taskbar, which looks a little out of place:
   > ![image](https://user-images.githubusercontent.com/22714352/215316186-39bc631e-467c-4d97-93a4-48451e2b84e1.png)
   > And if I reduce the size then the text doesn't look crisp anymore:
   > ![image](https://user-images.githubusercontent.com/22714352/215316189-564b547c-53b6-4605-a097-b4aec23e9ad5.png)
   > I think being able to choose font size rather than a % scaling would have a sharper result.

**Comment by emvaized** [comment]
I think the height changes because the width of text changes, when we compare "-9" and "-10" (one more digit added). Probably a "stretch" text setting was intended to solve this?

**Comment by FelixdelasPozas** [comment]
Hi @VadimAspirin and @emvaized . 

The application tries to maximize the size of the text to fill all the area available for it (a tray icon area) so less characters will make the text bigger automatically. It doesn't use a font size at all because the idea is to make the text as big as possible to be able to read it just with a quick glance at the taskbar. 

Letting the user set the font size means that (probably) you'll have to change it when the inclusion of a new character crops the border of a number because all the text won't fit the area. I tried to avoid that with the current implementation.

Regards,
Félix.

**Comment by rr1231234** [comment]
Icon size scaling is very inconsistent. At the moment it looks good at 70% for me but after a while it's displayed like 90% even though the setting is still 70%.

**Comment by FelixdelasPozas** [comment]
Hi @rr1231234 

Can you post screenshots?

**Comment by rr1231234** [comment]
> Hi [@rr1231234](https://github.com/rr1231234)
> 
> Can you post screenshots?

Actually the size scaling is bugged when it's 21 degrees.

**Comment by FelixdelasPozas** [comment]
Hi @rr1231234 

> Actually the size scaling is bugged when it's 21 degrees.

That is very oddly specific. Without knowing your settings like, for example, what font are you using, or posting some screenshots I can't do much. Please post screenshots of the image where it shows its bugged and the configuration you have. That will help me solve the issue.

Regards,
Félix.

**Comment by rr1231234** [comment]
> Hi [@rr1231234](https://github.com/rr1231234)
> 
> > Actually the size scaling is bugged when it's 21 degrees.
> 
> That is very oddly specific. Without knowing your settings like, for example, what font are you using, or posting some screenshots I can't do much. Please post screenshots of the image where it shows its bugged and the configuration you have. That will help me solve the issue.
> 
> Regards, Félix.

Sorry for the lack of info in the prev comments. My icon setting is like [this](https://imgur.com/a/bASvFDw), other tabs are in default state. 

Today after a long time observation I can be sure this is the problem: Whenever the temperature unit is reached 11 or 21 or 31 or 41 (**number 1 is the second digit**) the unit size will become bigger like at 100% scaling, but when it reaches 12 or 23 or 34... (**number 1 is not the second digit**) the size will become normal at 81% scaling as I set it.

This is a very tricky and odd scaling bug.

Thank you for your time.

**Comment by rr1231234** [comment]
You're right, after changing to mono-font it stays at 81% no matter what but the look of the font wont match the system haha

**Comment by FelixdelasPozas** [comment]
Yeah. But I think the icon is very customizable so with a little bit of tweaking one can find something that is readable and confortable. I'll close the issue as solved, but I'll think a little bit more about it because you're not the only one bothered with the change in size.

Regards, 
Félix.
